hydra-cardano-api/src/Hydra/Cardano/Api.hs

```diff
@@ -706,3 +706,15 @@ pattern ScriptWitness scriptWitnessInCtx scriptWitness <-
   Cardano.Api.ScriptWitness scriptWitnessInCtx scriptWitness
   where
     ScriptWitness = Cardano.Api.ScriptWitness
+
+-- | Sign transaction using the provided secret key
+-- It only works for tx not containing scripts.
+-- You can't sign a script utxo with this.
+signTx ::
+  SigningKey PaymentKey ->
+  Tx ->
+  Tx
+signTx signingKey (Tx body wits) =
+  makeSignedTransaction (witness : wits) body
+ where
+  witness = makeShelleyKeyWitness body (WitnessPaymentKey signingKey)
```

Comment:
Should: follow the organization guidelines in the `hydra-cardano-api/README.md` and move this to `Hydra.Cardano.Api.Tx` module, with a re-export in this module here.

Reply:
https://github.com/input-output-hk/hydra/commit/b62f839a1e70998e04f059eba6050e5a0005131a